client/src/App.js

```diff
@@ -1,62 +1,35 @@
-import React, { useState } from "react";
+import React from "react";
 import { MuiThemeProvider } from "@material-ui/core";
 import { BrowserRouter, Route, Redirect, Switch } from "react-router-dom";
-
 import { theme } from "./themes/theme";
-import { AuthContext } from "./context/auth-context";
-
 import Login from "./pages/Login";
 import Signup from "./pages/Signup";
 import CustomerProfile from "./pages/CustomerProfile";
 import NavBar from "./components/NavBar";
-
 import "./App.css";
+import { ContextProvider } from "./Context/UserContext";
```

Comment:
The folder name is context (lowercase), so the compile is breaking

Reply:
I had the folder set up as Context locally, I will change this to lowercase context